provisioner/terraform/otelenv_test.go

```diff
@@ -0,0 +1,63 @@
+package terraform // nolint:testpackage
```

Comment:
file should be named `otelenv_internal_test.go` then you don't need this `nolint`